src/components/RecentSubmission.js

```diff
@@ -3,16 +3,25 @@ import PropTypes from 'prop-types';
 import './RecentSubmission.css';
 
 const RecentSubmission = (props) => {
+
+  const hideRecentSubmission = () => {
+    if (props.submission !== undefined) {
+      return (
+        [<h3>The Most Recent Submission</h3>,
+        <p className="RecentSubmission__submission">{ props.submission }</p>]
```

Comment:
You should wrap all of this in a `div` tag or something and drop the array or give each element a `key` attribute.  This is why you're getting a warning.